packages/common/test/core/updateHubEntity.test.ts

```diff
@@ -0,0 +1,67 @@
+import { IArcGISContext } from "../../src/ArcGISContext";
+import { updateHubEntity } from "../../src/core/updateHubEntity";
```

Comment:
Please import everything from `../../src` (in tests) unless you are testing internal fns, or need to spy on a module. This will ensure that the fns are exported as expected.